pkg/sqlqueries/SQL/analytics/number_of_followers_per_model.sql

```diff
@@ -0,0 +1,9 @@
+/* Number of Followers By Model */
+SELECT
+    model_plan.model_name,
```

Comment:
Would you mind adding `model_plan.id` here as well like we do in the other queries?

Reply:
Thanks @ClayBenson94 ! Added